src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -979,10 +979,34 @@ export default function ({
     context.token = ret.token;
     context.otherTokenValues = ret.otherTokenValues;
     context.urlTokenPath = ret.urlTokenPath;
+
     const components = getTopLevelUrlCompleteComponents(context.method);
-    populateContext(ret.urlTokenPath, context, editor, true, components);
+    const { tokenPath, predicate } = (() => {
+      const lastUrlTokenPath =
+        Array.isArray(context.urlTokenPath) && context.urlTokenPath.length !== 0
+          ? context.urlTokenPath[context.urlTokenPath.length - 1]
+          : null;
+      if (
+        Array.isArray(lastUrlTokenPath) &&
+        !_.find(
+          components,
+          (c) => c instanceof ConstantComponent && _.find(lastUrlTokenPath, (p) => c.name === p)
+        )
+      ) {
+        return {
+          tokenPath: context.urlTokenPath.slice(0, -1),
+          predicate: (term) => term.meta === 'index',
```

Comment:
@yuliacech Thank you for your comment. But no good ideas came to mind.

I couldn't move the code to `IndexAutocompleteComponent` because of the necessity to look back at tokens before the comma. For example, `GET /a/b/c,d,` requires checking `c` and `d`. Further indices should be suggested only if both `c` and `d` are index-ish. Otherwise, no suggestion is preferable, like the current behavior.

I might be able to move the entire logic to `populateContext()` somehow, but I didn't do that because touching `engine.js` didn't seem a good idea.

I have added comments to clarify this point instead. I don't like the code volume of `addPathAutoCompleteSetToContext()` ballooning, though.